package/network/services/hostapd/patches/010-mesh-Allow-DFS-channels-to-be-selected-if-dfs-is-ena.patch

```diff
@@ -1,29 +1,36 @@
-From 8de8cd8380af0c43d4fde67a668d79ef73b26b26 Mon Sep 17 00:00:00 2001
-From: Peter Oh <peter.oh@bowerswilkins.com>
-Date: Tue, 30 Jun 2020 14:18:58 +0200
-Subject: [PATCH 10/19] mesh: Allow DFS channels to be selected if dfs is
- enabled
-
-Note: DFS is assumed to be usable if a country code has been set
-
-Signed-off-by: Benjamin Berg <benjamin@sipsolutions.net>
-Signed-off-by: Peter Oh <peter.oh@bowerswilkins.com>
----
- wpa_supplicant/wpa_supplicant.c | 25 +++++++++++++++++++------
- 1 file changed, 19 insertions(+), 6 deletions(-)
-
---- a/wpa_supplicant/wpa_supplicant.c
-+++ b/wpa_supplicant/wpa_supplicant.c
-@@ -2395,6 +2395,8 @@ void ibss_mesh_setup_freq(struct wpa_sup
+--- a/wpa_supplicant/wpa_supplicant.c	2022-01-16 21:51:29.000000000 +0100
```

Comment:
Why there was removed the git header with details like author, subject, commit message, etc.?

Reply:
because it's no longer Peter Oh who wrote this patch, It's me performing diff -u wpa_supplicant.c wpa_supplicant.c.orig after failing to apply 010-mesh-... due to changes in code in hostapd 2.10

Reply:
That's why you can add Signed-off-by to the commit message, and above in these [] brackets add that it was rebased to hostapd 2.10 version. We can not lose such details of what the patch does w/o looking into the source code.

Reply:
And if the patch was sent to upstream, which I did not check it, you can suggest your changes there.